java-manta-client-unshaded/src/test/java/com/joyent/manta/util/ContinuingInputStreamTest.java

```diff
@@ -364,7 +364,7 @@ private static void addToParamListIfUnseen(final Set<Integer> hashcodes,
         paramList.add(params);
     }
 
```

Comment:
Boolean Value ```finished``` at L342 is always false, empty ```catch``` block at L331, ```if, if-else ```conditions in L258 & L260, L267 & L269 already covered in subsequent conditions,

Reply:
Unused ```Exceptions``` thrown in L193, L149, L130, L73.